conans/cli/commands/list.py

```diff
@@ -34,14 +35,54 @@ def list_recipes_cli_formatter(info):
             cli_out_write("{}{}".format(" " * (indentation * 2), reference), reference_color)
 
 
-def list_recipes_json_formatter(info):
+def _list_revisions_cli_formatter(results, is_package=False):
+    for remote_results in results:
+        if remote_results.get("error"):
+            # TODO: Handle errors
+            return
+
+        if not remote_results.get("remote"):
+            cli_out_write("Local Cache:", remote_color)
+        else:
+            cli_out_write("{}:".format(remote_results["remote"]), remote_color)
+
+        tab_space = " " * indentation
+        if not remote_results.get("results"):
+            ref_type = "packages" if is_package else "recipes"
+            cli_out_write(f"{tab_space}There are no matching {ref_type}")
```

Comment:
I'd rather avoid mixing f-strings and `format()` in the same file

Reply:
Completely right, I did not see the other `format`, my bad 😅